djangorestframework_camel_case/util.py

```diff
@@ -20,6 +22,11 @@ def camelize(data):
         for i in range(len(data)):
             data[i] = camelize(data[i])
         return data
+    if isinstance(data, QuerySet):
+        new_list = []
+        for i in range(len(data)):
+            new_list.append(camelize(data[i]))
+        return new_list
```

Comment:
I see now that this line was inspired by line 21/23 above, where the existing list is modified in place (which is broken for tuples, debatable for lists and already discussed in other tickets).